assets/src/application/ui/display/Popover/index.tsx

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
+import classNames from 'classnames';
 
 import PopoverAdapter, { PopoverProps } from '@infraUI/display/Popover';
 
 import './style.scss';
 
-const Popover: React.FC<PopoverProps> = (props) => <PopoverAdapter {...props} contentClassName="ee-popover__content" />;
+const Popover: React.FC<PopoverProps> = (props) => {
+	const className = classNames('ee-popover', props.className);
+	return (
+		<div className={className}>
+			<PopoverAdapter {...props} contentClassName='ee-popover__content' />
+		</div>
+	);
+};
```

Comment:
need this div wrapper in order to style popovers properly, so extracted this from the timezone info component into here instead of duplicating it again where I needed it